tests/validation/tests/v3_api/test_logging_e2e.py

```diff
@@ -0,0 +1,224 @@
+import pytest
```

Comment:
Alphabetize the imports

Reply:
I just updated this, please check again. 👍